src/main/java/org/testng/TestRunner.java

```diff
@@ -658,6 +659,11 @@ public long getTimeOut() {
         return 0;
       }
 
+      @Override
+      public void setLatch(CountDownLatch latch) {
+
```

Comment:
Maybe copy/paste the comment above saying this method is intentionally empty (which I really like).